test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -102,9 +104,33 @@ private void Context_Log(object sender, LoggingEventArgs e)
             //throw new NotImplementedException();
         }
 
-        [Fact]
-        public void AutoFitRegressionTest()
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("ar-SA")]
+        [InlineData("pl-PL")]
+        public void AutoFitRegressionTest(string culture)
         {
+            var originalCulture = Thread.CurrentThread.CurrentCulture;
+            Thread.CurrentThread.CurrentCulture = new CultureInfo(culture);
+
+            uint experimentTime = 0;
+
+            if (culture == "ar-SA")
+            {
+                // If users run AutoML with a different local, sometimes
+                // the sweeper encounters problems when parsing some strings.
+                // So testing in another culture is necessary.
+                // Furthermore, these issues might only occur after several
```

Comment:
We have a more specific value:
```suggestion
                // Furthermore, these issues might only occur after ~70
```

Reply:
Thanks.
#Resolved